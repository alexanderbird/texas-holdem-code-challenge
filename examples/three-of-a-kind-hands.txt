2H 4C 6S 8D TC
Chip 4H 4S JH
Kira TH TS 3C
Cheater TH TS JC
