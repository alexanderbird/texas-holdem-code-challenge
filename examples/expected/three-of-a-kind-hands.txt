1 Cheater Three of a Kind 10 (kicker Jack)
2 Kira Three of a Kind 10 (kicker 8)
3 Chip Three of a Kind 4
